src/content/contribute/writing-a-loader.md

```diff
@@ -333,15 +333,15 @@ export default (fixture, options = {}) => {
         test: /\.txt$/,
         use: {
           loader: path.resolve(__dirname, '../src/loader.js'),
-          options: {
-            name: 'Alice'
-          }
+          options,
```

Comment:
I think it's bit nicer to pass options from your tests so this is more reusable.